### PR TITLE
feat: add implementation of splitSection function

### LINKED ISSUE #20: Create implementation of splitSection function
Please, create an implementation for splitSection function. This function should slice a section into array of options. Every element of array is string (option with description).

### PR DESCRIPTION
**Pull request by ifedchankau**
Add implementation of splitSection function.

Closes #20

### CODE REVIEW

File: src/handle.js [hunk 1]
```diff
@@ -19,8 +19,17 @@ const parseSection = (sectionName, help) => {
 };
 
 const splitSection = (section) => {
-    //TODO: create implementation
     let split = [];
+    const regularExp = "\n[ \t]*";
+    let splitTmp = (section.split(regularExp)).slice(1);
```
Comment (xferra) [inline src/handle.js]: Unnecessary brackets.

File: src/handle.js [hunk 2]
```diff
@@ -19,8 +19,17 @@ const parseSection = (sectionName, help) => {
...
+    for (let cnt = splitTmp.length, i = 0, j = 0; i < cnt; i++, j++) {
```
Comment (xferra) [inline src/handle.js]: What is the idea behind? Its' quite hard to understand such code without comments.

**Review by romanrostislavovich — APPROVED** [review]
+1

**Pull request merged**